lib/webauthn/authenticator_data/attested_credential_data.rb

```diff
@@ -1,16 +1,27 @@
 # frozen_string_literal: true
 
+require "bindata"
 require "cose/key"
+require "webauthn/error"
 
 module WebAuthn
-  class AuthenticatorData
-    class AttestedCredentialData
+  class AuthenticatorData < BinData::Record
+    class AttestedCredentialData < BinData::Record
+      class AttestedCredentialDataFormatError < WebAuthn::Error; end
+
       AAGUID_LENGTH = 16
       ZEROED_AAGUID = 0.chr * AAGUID_LENGTH
 
       ID_LENGTH_LENGTH = 2
 
-      UINT16_BIG_ENDIAN_FORMAT = "n*"
+      endian :big
+
+      count_bytes_remaining :data_length
```

Comment:
Is this `data_length` used anywhere?

Reply:
It's not. It a remnant from before removing the length validation.